Inventor_API/Models/FoodItem.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Inventor_API.Models
+{
+    public class FoodItem
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
```

Comment:
Why do you have description and not quantity in this model?
Please refer to the ERD of the project when creating future models

Don't forget to include the list of categories